src/consts.js

```diff
@@ -10,7 +10,7 @@ if (REGION === undefined) {
 
 module.exports.REGION = REGION;
 
-module.exports.TRACE_COLLECTOR_URL = `https://${REGION}.tc.epsagon.com`;
+module.exports.TRACE_COLLECTOR_URL = `http://${REGION}.tc.epsagon.com`;
```

Comment:
can't we make the default https and make the if in the previous file check `(configData.useSSL === false)`?